tensorrt_llm/sampling_params.py

```diff
@@ -321,6 +324,11 @@ def _validate(self):
                 f"under the greedy decoding."
             )
 
+        if self.logprobs_mode not in ["raw", "processed"]:
```

Comment:
nit: since this flag is for PyTorch backend only, it'd be better that we also add a guard for it.